roles/common/tasks/RedHat.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Install base packages
+  yum: name={{item}} state=installed
+  with_items:
+    - curl
+    - unzip
+    - ca-certificates
+
+- name: check kernel version
+  shell: uname -r| tr -d '.|-' |cut -c 1-7
```

Comment:
It seems like the goal is to upgrade the kernel firmware, so suggest we do this by first [enable the centosplus repo](https://wiki.centos.org/AdditionalResources/Repositories/CentOSPlus) which seems to be the official source of the kernel-firmware:

```
- name: Add repository
  yumrepo:
    name: centosplus
    baseurl: http://mirror.centos.org/centos/$releasever/centosplus/$basearch
    mirrorlist: http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=centosplus
    gpgcheck=yes
    includepkgs=kernel-firmware-*
```

and then use the `yum` task to force installation of the version listed below